lib/api/users/user.ts

```diff
@@ -0,0 +1,27 @@
+import { db} from "../../db";
+
+export function getlockers() {
```

Comment:
This function is already in `admin.ts` - do you think it needs to be in both? or maybe it should be in a separate util module? something to think about

Reply:
Since you're accessing the `lockers` table I would keep it out of users.